templates/aurora_mysql-main.template.yaml

```diff
@@ -318,12 +318,8 @@ Parameters:
       - 'Aurora-MySQL5.7-2.09.0'
       - 'Aurora-MySQL5.7-2.09.1'
       - 'Aurora-MySQL5.7-2.09.2'
-      - 'Aurora-MySQL5.7-2.08.0'
-      - 'Aurora-MySQL5.7-2.08.1'
-      - 'Aurora-MySQL5.7-2.08.2'
-      - 'Aurora-MySQL5.7-2.09.0'
   DBEngineMode:
-    Description: "Engine mode of the cluster. With provisioned databases (the default), you provision and manage the server instance sizes. Aurora Serverless, Global Database, and Multi-Master are available for Aurora-MySQL5.6.10a. Aurora Parallel Query is available for Aurora MySQL5.6-x versions. For descriptions, see https://aws.amazon.com/rds/aurora/mysql-features/."
+    Description: "Engine mode of the cluster. With provisioned databases (the default), you provision and manage the server instance sizes. For descriptions, see https://aws.amazon.com/rds/aurora/mysql-features/."
```

Comment:
@catcharbind Please add "For descriptions _of the other options_, see..."

Reply:
Fixed and pointed to https://aws.amazon.com/rds/aurora/faqs/ instead.

Reply:
@catcharbind I'm glad that we now point to a page (the FAQ) that describes all the engine-mode options. Thanks.